### DISCUSSION TITLE
Seeking Guidance: Beginner Contributor Looking for Good First Issues

### DISCUSSION THREAD

**Discussion by mukesh-mahato** [CollaboraOnline/online#6393] [Q&A] [answered]
Hello everyone! I'm thrilled to be part of the Collabora Online community and eager to contribute to this amazing project. As a beginner, I'm seeking guidance and suggestions from the more experienced members of the community. I would greatly appreciate it if you could help me identify some good first issues that I can work on to get started with contributing. Let's collaborate and empower new contributors to make a meaningful impact!
Thank you in advance for your suggestions, guidance, and support! I'm excited to embark on this journey of contributing to Collabora Online and look forward to making a positive contribution to the project.

**Comment by Darshan-upadhyay1110** [comment, marked as answer]
Hii @mukeshmahato18 it's great that you want to contribute to online. i suggest first set up the COOL build in your machine if you are on linux or else you can use gitpod as well.

then to start with code base you can go to issues section and filter the issues with `easy hack` which are good to start with. 

https://collaboraonline.github.io/post/build-code/#build-code-on-ubuntu

thanks
